tests/utils.js

```diff
@@ -1,17 +1,20 @@
-require('babel-polyfill');
-require('co-mocha');
-
-let expect = require('chai').expect;
-let sinon = require('sinon');
-let _ = require('lodash');
-
 describe('Utils', () => {
-    let utils;
+    let _, sinon, expect, utils;
 
     before(() => {
-        utils = require('../dist/utils.js');
+        _ = require('lodash');
+        sinon = require('sinon');
         let chai = require('chai');
         chai.use(require('chai-sinon'));
+        expect = require('chai').expect;
+        utils = require('../dist/utils.js');
+        sinon = require('sinon');
```

Comment:
Hi @rhoggSugarcrm I honestly don't remember. If it works without it, we can remove it.